packages/mobile/src/utils/otaTranslations.ts

```diff
@@ -0,0 +1,15 @@
+import { Resource, ResourceLanguage } from 'i18next'
+import * as RNFS from 'react-native-fs'
+import { OTA_TRANSLATIONS_FILEPATH } from 'src/config'
+
+export function saveOtaTranslations(language: string, translations: ResourceLanguage) {
+  return RNFS.writeFile(OTA_TRANSLATIONS_FILEPATH, JSON.stringify({ [language]: translations }))
+}
+
+export async function getOtaTranslations() {
```

Comment:
if `save` takes `language`, shouldn't `get` take a `language` to avoid exposing the implementation details of the cache?  i.e., the `{ [language]: ... }` structure defined in `save`.

Reply:
i made the `saveOtaTranslations(resource: Resource)` -> `getOtaTranslations(): Resource` change we talked about, it does look much better 😀